src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/handlers/ReconfigurationCaseEventHandlerTest.java

```diff
@@ -507,4 +497,17 @@ private EvaluateDmnRequest buildEvaluateUpdateDmnRequest() {
         return new EvaluateDmnRequest(variables);
     }
 
+    private void assertConsoleOutputHasMessages(CapturedOutput output) {
+        await().ignoreException(Exception.class)
+            .pollInterval(100, MILLISECONDS)
+            .atMost(5, SECONDS)
+            .untilAsserted(() -> {
+                Assertions.assertTrue(output.getOut().contains(RECONFIGURATION_EVENT_INFORMATION_LOG));
+                Assertions.assertTrue(output.getOut().contains(SEND_RECONFIGURATION_REQUEST_LOG));
+                Assertions.assertTrue(output.getOut().contains(RECONFIGURATION_COMPLETED_LOG));
+            });
```

Comment:
This looks a lot better but are there any other tests which would benefit from the same improvement?  Would be good to catch them all at once.

Reply:
👍